src/lib/RosNode.js

```diff
@@ -299,6 +299,41 @@ class RosNode extends EventEmitter {
     return this._masterApi.getSystemState(this._nodeName, options);
   }
 
+  /** Services currently do not declare their type with the master, so instead
+    we probe the service for its headers. Just like in
+  https://github.com/ros/ros_comm/blob/6292d54dc14395531bffb2e165f3954fb0ef2c34/tools/rosservice/src/rosservice/__init__.py#L94-L98
+  */
+  getServiceType(serviceName) {
```

Comment:
should this be called `getServiceHeaders`? `ros_comm` has both and `getServiceType` is a wrapper around `getServiceHeaders`

Reply:
The different between the two in ros_comm seems to be that `get_service_type` takes just the service name as argument and looks up the name whereas `get_service_headers` requires the service URI as argument. So in that analogy `getServiceType` would seem to be the right name to use here.

Reply:
I was mostly looking at the return type - `get_service_headers` returns headers (like this function) while `get_service_type` returns only the type field of the header.

Reply:
Good point, yes. In that case I agree. Done.